app/src/main/res/layout/add_profile_activity.xml

```diff
@@ -250,4 +241,4 @@
         android:background="@drawable/toolbar_drop_shadow" />
     </FrameLayout>
   </LinearLayout>
-</layout>
+</layout>
```

Comment:
EOF is missing, add one blank line at the end.

Reply:
Done.